pilot/test/integration/driver.go

```diff
@@ -203,14 +203,8 @@ func runTests(envs ...infra) {
 		}
 
 		tests := []test{
-			&http{infra: &istio},
-			&grpc{infra: &istio},
-			&tcp{infra: &istio},
-			&headless{infra: &istio},
-			&ingress{infra: &istio},
 			&egressRules{infra: &istio},
-			&routing{infra: &istio},
-			&zipkin{infra: &istio},
```

Comment:
You know you can specify the test you want to run as part of the command line right? :).

Reply:
Oops :) No, I do not know that. Is it documented somewhere?

Reply:
driver.go

Reply:
ah, I see, testtype flag